src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age !== undefined
+    && (
+    <p className="Person__age">
+        {`I am ${person.age}`}
+    </p>
+    )}
+    <p className="Person__partner">
+      {!person.isMarried && <p>I am not married</p> }
+      {(person.isMarried && person.sex === 'm')
+        && `${person.partnerName} is my wife`}
+      {(person.isMarried && person.sex === 'f')
+        && `${person.partnerName} is my husband`
+      }
```

Comment:
**person.isMarried**  this condition is duplicated, let's organize your code
```suggestion
     {person.isMarried && (
        {person.sex === 'm' && (
          `${person.partnerName} is my wife`
        )}
       {person.sex === 'f' && (
        `${person.partnerName} is my husband`
       )}
     )}
```

Reply:
it was some linter errors so fixed this part with another way